src/System.Security.Cryptography.Cng/src/Microsoft/Win32/SafeHandles/NCryptSafeHandles.cs

```diff
@@ -53,42 +56,46 @@ private enum OwnershipState {
             Holder
         }
 
-        private OwnershipState m_ownershipState;
+        private OwnershipState _ownershipState;
 
         /// <summary>
         ///     If the handle is a Duplicate, this points at the safe handle which actually owns the native handle.
         /// </summary>
-        private SafeNCryptHandle m_holder;
+        private SafeNCryptHandle _holder;
 
-        protected SafeNCryptHandle() : base(IntPtr.Zero, true) {
+        protected SafeNCryptHandle() : base(IntPtr.Zero, true)
+        {
             return;
         }
 
         /// <summary>
-        ///     Wrapper for the m_holder field which ensures that we're in a consistent state
+        ///     Wrapper for the _holder field which ensures that we're in a consistent state
         /// </summary>
-        private SafeNCryptHandle Holder {
-            get {
-                Contract.Requires((m_ownershipState == OwnershipState.Duplicate && m_holder != null) ||
-                                  (m_ownershipState != OwnershipState.Duplicate && m_holder == null));
-                Contract.Requires(m_holder == null || m_holder.m_ownershipState == OwnershipState.Holder);
+        private SafeNCryptHandle Holder
+        {
+            get
+            {
+                Contract.Requires((_ownershipState == OwnershipState.Duplicate && _holder != null) ||
```

Comment:
We're not using the contract rewriter, nor are we compiling with CONTRACTS_FULL.  At least for now, should we make all of these Contract.Requres into Debug.Asserts?